operators/src/processing/point_in_polygon.rs

```diff
@@ -87,8 +87,13 @@ impl VectorOperator for PointInPolygonFilter {
             }
         );
 
+        let in_desc = points.result_descriptor().clone();
+
+        let mut out_desc = in_desc;
+        out_desc.bbox = polygons.result_descriptor().bbox;
```

Comment:
I think it's more complicated because of multi point features. it suffices that one point is inside for the whole feature to be kept 
![grafik](https://user-images.githubusercontent.com/10283892/173330995-2d1f4bd5-f7e8-4a81-95d2-be3331fad9ed.png)

Reply:
Okay, we could say union or somewhat mangle around with it. 

But still, it cannot be larger than the points bbox.
And it cannot be larger than the polygon's bbox.

But what are the semantic here?
What do we want to achieve with the bbox?

Reply:
> Okay, we could say union or somewhat mangle around with it.

Imho we just have to take the points bbox. We cannot be sure that any points will be filtered out as I stated above. But we CAN be sure that no new points are introduced.

So as long as the boxes of points and polygons intersect at all, we have to take the points bbox as the output descriptor. I am not sure what to do in the case they don't intersect but for now I guess we still take the points bbox

> But what are the semantic here?
What do we want to achieve with the bbox?

they are the bounds of the (derived) dataset and say where there is potentially data and that outside of it there will never be data. We want to use it for optimizations and for navigation in the UI.

Reply:
Hm okay, I guess we can go with that.
But then you have to change it, because currently it uses the polygon's bbox.

Reply:
b1cb6df875cb6dc0cb7ef232b52e7ddf33bc2700